ui/packages/client/src/client.ts

```diff
@@ -350,12 +352,16 @@ function skip_queue(id: number, config: Config) {
 
 async function resolve_config(endpoint?: string): Promise<Config> {
 	if (window.gradio_config && location.origin !== "http://localhost:9876") {
-		return { ...window.gradio_config, root: endpoint };
+		const path = window.gradio_config.root;
```

Comment:
Shouldn't this part be changed to `window.location.pathname`? It seems that the current value of the root is being included instead of the path variable, which should contain the path.